src/Datadog.Trace.ClrProfiler.Managed/ScopeFactory.cs

```diff
@@ -41,32 +43,36 @@ public static Scope CreateOutboundHttpScope(Tracer tracer, string httpMethod, Ur
 
                 if (parent != null &&
                     parent.Type == SpanTypes.Http &&
-                    parent.GetTag(Tags.HttpMethod).Equals(httpMethod, StringComparison.OrdinalIgnoreCase) &&
-                    parent.GetTag(Tags.HttpUrl).Equals(UriHelpers.CleanUri(requestUri, removeScheme: false, tryRemoveIds: false), StringComparison.OrdinalIgnoreCase))
+                    parent.GetTag(Tags.InstrumentationName) != null)
                 {
                     // we are already instrumenting this,
                     // don't instrument nested methods that belong to the same stacktrace
                     // e.g. HttpClientHandler.SendAsync() -> SocketsHttpHandler.SendAsync()
                     return null;
                 }
 
+                string httpUrl = requestUri != null ? UriHelpers.CleanUri(requestUri, removeScheme: false, tryRemoveIds: false) : null;
```

Comment:
The previous code had `tryRemoveIds: true` for the resource name and `tryRemoveIds: false` for the tag. I'm guessing this change was not intentional.

Reply:
and `removeScheme` was also different

Reply:
Good catch, I didn't intend to change behavior. I'm writing up a fix and trying to write a regression test on this behavior as well

Reply:
Fixed in 9c4d7c0026db1b7ef22399cf788091b2cb6e7cea with some unit-tests backing it up 😄